frontend/src/entry/model/ModelImageList.tsx

```diff
@@ -22,12 +22,26 @@ type PartialModelImageListProps =
 
 type ModelImageListProps = {
   model: EntryInterface
+  onRegistryError: (value: boolean) => void
   readOnly?: boolean
 } & PartialModelImageListProps
 
-export default function ModelImageList({ model, value, onChange, readOnly = false, multiple }: ModelImageListProps) {
+export default function ModelImageList({
+  model,
+  value,
+  onChange,
+  onRegistryError,
+  readOnly = false,
+  multiple,
+}: ModelImageListProps) {
   const { modelImages, isModelImagesLoading, isModelImagesError } = useGetModelImages(model.id)
 
+  const handleRegistryError = useCallback((value: boolean) => onRegistryError(value), [onRegistryError])
```

Comment:
This callback doesn't prevent infinite rendering because it's dependent on `onRegistryError`. You can change the below `useEffect` to be like it was before:
```
  useEffect(() => {
    onRegistryError(!!isModelImagesError)
  }, [isModelImagesError, handleRegistryError])
```

For context, the reason why the `useCallback`s in the release and inferencing form components work is because they are only dependent on a `useState` setter function, which React guarantees is a stable function (also why it makes no difference as to whether or not you include setter functions in the dependency array)